tools/build_pak_files.py

```diff
@@ -57,21 +51,46 @@
     # once we know the platform, set our cooked dir
     unreal_project_cooked_dir = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", platform + "NoEditor"))
 
-    if spear.path_exists(unreal_project_content_shared_dir):
-        print(f"[SPEAR | build_pak_files.py] File or directory or symlink exists, removing: {unreal_project_content_shared_dir}")
-        spear.remove_path(unreal_project_content_shared_dir)
+    # We do not want to use os.path.realpath(...) here, because that will resolve to the directory inside the user's Perforce workspace.
+    # Instead, we want this path to refer to the symlinked version inside the user's unreal project directory.
+    unreal_project_content_shared_dir = os.path.join(unreal_project_content_dir, "Shared")
+    unreal_project_content_scenes_dir = os.path.join(unreal_project_content_dir, "Scenes")
+
+    # scenario wherein we do not want to create symlinks
+    if args.skip_create_symlinks:
+        assert args.scene_ids is not None   # user should input the scene id
```

Comment:
no need for these comments because the code is already self-documenting:

```
 # user should input the scene id
# shared dir should already exist
# scenes dir should already exist
```

Reply:
Done